content/blog/2020-02-10-introduction-to-machine-learning-with-the-tidyverse/index.Rmarkdown

```diff
@@ -0,0 +1,191 @@
+    + `fit_data()` = [`parsnip::fit()`](https://tidymodels.github.io/parsnip/reference/fit.html)
+    + `fit_split()` = [`tune::last_fit()`](https://tidymodels.github.io/tune/reference/last_fit.html)
+
+
+## Teaching Infrastructure
+
++ **RStudio Server Pro:** Our [RStudio Server Pro](https://rstudio.com/products/rstudio-server-pro/) workspaces used Amazon compute optimized [`c5.large` instances](https://aws.amazon.com/ec2/instance-types/) with 2 vCPUs and 4 GiB memory for each learner.
+
++ **Slides:** I used the [xaringan package](https://github.com/yihui/xaringan) to build all my slides in R Markdown. For a tutorial, you can see my rstudio::conf(2019) workshop slides [here](https://arm.rbind.io/slides/xaringan.html#1). I also made heavy use of the [countdown package](https://pkg.garrickadenbuie.com/countdown/#1) to create the "Your Turn" timers `r emo::ji("timer")`.
+
++ **Workshop website:** I used the [blogdown R package](https://bookdown.org/yihui/blogdown/) to build the [website](https://conf20-intro-ml.netlify.com), with the [Hugo academic theme](https://themes.gohugo.io/academic/) with a custom CSS designed by [Desirée De Leon](http://desiree.rbind.io/). If you want to re-use my workshop website (you'll need [GitHub](https://github.com/) and [Netlify](https://www.netlify.com/) accounts), click on the button below `r emo::ji("rocket")`: <br><br>
+[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/rstudio-conf-2020/intro-to-ml-tidy)
+
+## Thanks
+
+I sincerely enjoyed developing this workshop with [Garrett](https://twitter.com/StatGarrett), getting a chance to work closely with [Max Kuhn](https://twitter.com/topepos) and [Davis Vaughan](https://twitter.com/dvaughan32) of the tidymodels team (now including the inimitable [Julia Silge](https://juliasilge.com/)!), and having the opportunity to introduce a new cohort of R and tidyverse users to [tidymodels](https://github.com/tidymodels). I hope the materials we developed are useful to learners and other teachers too- if they are, [please let me know](https://twitter.com/apreshill), I'd love to hear about it.
+
+Happy predictive modeling!
+
+
+
+[00-slides]: https://conf20-intro-ml.netlify.com/materials/00-welcome/
```

Comment:
I know it's a quirk, but I always put all of my links in a table at the end now rather than embedding any of them directly...